src/index.html

```diff
@@ -9,10 +13,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The stylesheet link should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in HTML.